openapi/openapi.yml

```
@@ -54,6 +54,66 @@ paths:
       tags:
         - post
       summary: ''
+    post:
+      description: Create a new post.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                title:
+                  type: string
+                body:
+                  type: string
+                user_id:
+                  type: integer
+              required:
+                - title
+                - body
+                - user_id
+              example:
+                title: "Hello, World!"
+                body: "This is my first post."
+                user_id: 0
```

以下みたいな感じでまとめていきたい！
他の書き方を参考にしてみてほしい。

```suggestion
          application/json:
            schema:
              $ref: '#/components/schemas/CreatePostRequest'
            example:
              default:
                value:
                  title: "Hello, World!"
                  body: "This is my first post."
                  user_id: 0
```